os_faults/drivers/tcpcloud.py

```diff
@@ -203,13 +216,13 @@ def verify(self):
 
     def _get_cloud_hosts(self):
         if not self.cached_cloud_hosts:
-            cmd = "salt -E '(ctl*|cmp*)' network.interfaces --out=yaml"
+            cmd = "salt -E '(infra*)' network.interfaces --out=yaml"
             result = self.execute_on_master_node({'command': cmd})
             stdout = result[0].payload['stdout']
             for fqdn, net_data in yaml.load(stdout).items():
                 host = node_collection.Host(
-                    ip=net_data['eth0']['inet'][0]['address'],
-                    mac=net_data['eth0']['hwaddr'],
+                    ip=net_data['eth1']['inet'][0]['address'],
```

Comment:
I am not sure if tcpcloud requires eth1. As we are using OSA deployed cloud we might want eth1 but not sure if this change would be relevant to tcpcloud config.  I would suggest not make any changes to tcpcloud driver.